doc/manual/rl-next/shebang-relative.md

```diff
@@ -0,0 +1,10 @@
+synopsis: ensure nix-shell shebang uses relative path
+prs: #5088
+description: {
+
+`nix-shell` shebangs use the script file's relative location to resolve relative paths to files passed as command line arguments, but expression arguments were still evaluated using the current working directory as a base path.
```

Comment:
```suggestion
`nix-shell` shebangs use the script file's relative location to resolve relative paths passed as command line arguments, but paths passed in expression arguments were still evaluated using the current working directory as a base path.
```

Reply:
It took me a moment to fully grasp what this means. A before and after example in the config documentation would be very helpful, but not blocking.